images/runtime/python/3.11/base.bullseye.Dockerfile

```diff
@@ -29,7 +29,7 @@ ADD build ${BUILD_DIR}
 RUN find ${IMAGES_DIR} -type f -iname "*.sh" -exec chmod +x {} \;
 RUN find ${BUILD_DIR} -type f -iname "*.sh" -exec chmod +x {} \;
 
-ENV PYTHON_VERSION 3.11.0
+ENV PYTHON_VERSION 3.11.8
```

Comment:
Are we sure we need to update this for base.bullseye Dockerfile also?

Reply:
Let me come back on this to you.

Reply:
We shouldn't need to update this, unless there has been an issue associated with this.

Reply:
I also realized from the code that template.dockerfile is copied to base.bullseye.Dockerfile

Reply:
But this should not create an issue as it would be overridden and it's fine to keep the parity like bookworm.